.github/workflows/codeql.yml

```diff
@@ -0,0 +1,48 @@
+name: CodeQL
+
+on:
+  pull_request:
+  workflow_dispatch:
+    branches: [ "master" ]
+
+jobs:
```

Comment:
I would add `if: github.repository == 'tarantool/tarantool'` to avoid running this inside forks. Example https://github.com/tarantool/tarantool/blob/master/.github/workflows/debug.yml#L37

Reply:
Looks fixed, thanks.